docs/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Changed
 - Use keywords from ps-constants in CVEorg collector (OSIDB-3694)
+- Added last_validated_dt field to AlertMixin and created_dt to alerts;
+  optimized alert handling by matching valid alerts via timestamps
+  and scheduling stale alert removal to reduce database locks. (OSIDB-3579)
```

Comment:
nitpick (feel free to ignore): The CHANGELOG should be primarily for the user (the developers have Git history) and I am not sure they could have an idea what those attribute names stand for :smile: So describing the reasons or effects of the change is more important - or sometimes the CHANGELOG record could be IMO fully omitted if we do some purely internal changes which do not affect the user or in some totally indirect way.

Reply:
Removed the cange as it does not add value to the users